src/app/pages/send/send-crypto-asset-form/form/btc/btc-send-form.tsx

```diff
@@ -42,6 +41,9 @@ export function BtcSendForm() {
     validationSchema,
   } = useBtcSendForm();
 
+  // with this enabled PageLayout re-renders in a loop
```

Comment:
Using `useUpdatePageHeaderContext` to update the title here causes a re-render loop in `PageLayout` . It seems to work well generally otherwise but is clearly not a good fit here